packages/components/Button/src/FAB/__snapshots__/FAB.test.tsx.snap

```diff
@@ -26,18 +26,11 @@ exports[`Custom FAB with no shadow(iOS) 1`] = `
   onStartShouldSetResponder={[Function]}
   style={
     Object {
-      "alignItems": "center",
-      "alignSelf": "flex-start",
+      "alignSelf": "baseline",
```

Comment:
The align-self change seems like an unintended change?

Reply:
Also all of these tests are meant to test iOS - you should check with Lynn if this is okay or if you need to be standing up separate tests for Android.

Reply:
Yeah the FAB tests specifically currently have the Platform check mocked out to always return 'ios', so they wouldn't be testing the ripple. I think you will need to add some of your own tests where the Platform check isn't returning 'ios' (either it should be mocked out to return 'android' or I think if the default if nothing is specified is actually to test Android). If we can keep the tests in the same file that seems better to me than splitting it into a FAB ios and FAB android test file, since I think we would want Shadow on Android at some point (not sure if we'd want ripple on iOS) but I don't have a strong opinion about this.

About the changes to the snapshot: I'm not sure if the changes are okay or not, but I can help you test that these changes haven't impacted the iOS fab in the tester - feel free to message me on teams.

Reply:
Hey @lyzhan7,
I don't think 
```
beforeAll(() => {
  jest.mock('react-native/Libraries/Utilities/Platform', () => ({
    OS: 'ios',
    select: () => null,
  }));
});
```
 is working as intended, I set the FAB.ios.tsx file to return null but the test still ran successfully! 
I think it is running with Android as platform, which also explains why the ripple addition caused this snapshot to change

Reply:
Hmm okay just to check, is `Platform.OS === 'android'` on line 83 in FABCore.tsx returning true? If it's returning true that's unexpected & I'll look into that. I see also in your code you've introduced `Platform.select({android:...}`. I'm not familiar with Platform.select, but I think the tests might need to mock out that method as well to keep all of the Android ripple stuff out of iOS (would guess that you add another jest.mock like under the current one, but it mocks out `select` instead of `ios` - something like that)